monkey/monkey_island/cc/next_ui/src/app/(auth)/login/page.tsx

```diff
@@ -64,85 +66,80 @@ const LoginPage = () => {
-                            flexDirection: 'column',
-                            alignItems: 'center'
-                        }}>
-                        <Avatar sx={{ m: 1, bgcolor: 'primary.main' }}>
-                            <LockOutlinedIcon />
-                        </Avatar>
-                        <Typography component="h1" variant="h5">
-                            Login
-                        </Typography>
-                        <Box
-                            component="form"
-                            onSubmit={handleSubmit}
-                            sx={{ mt: 1 }}>
-                            <TextField
-                                margin="normal"
-                                required
-                                fullWidth
-                                id="username"
-                                label="username"
-                                name="username"
-                                autoComplete="username"
-                                value={loginFormValues.username}
-                                onChange={handleLoginFormValueChange}
-                                autoFocus
-                            />
-                            <TextField
-                                margin="normal"
-                                required
-                                fullWidth
-                                name="password"
-                                label="Password"
-                                type="password"
-                                id="password"
-                                autoComplete="current-password"
-                                value={loginFormValues.password}
-                                onChange={handleLoginFormValueChange}
-                            />
-                            <FormControlLabel
-                                control={
-                                    <Checkbox
-                                        value="remember"
-                                        color="primary"
-                                    />
-                                }
-                                label="Remember me"
-                            />
-                            <Button
-                                type="submit"
-                                fullWidth
-                                variant="contained"
-                                sx={{ mt: 3, mb: 2 }}>
-                                Login
-                            </Button>
+            <Container component="main" maxWidth="xs" sx={containerStyle}>
+                <Stack direction="column" alignItems={'center'} spacing={2}>
+                    {renderFormCard()}
+                </Stack>
+            </Container>
+        );
+    };
+
+    const renderFormCard = () => {
+        return (
+            <Card variant="outlined" sx={cardStyle(theme)}>
+                <MonkeyLogo sx={{ height: '50px' }} />
```

Comment:
Can we move this style to `login/style.ts`?

Reply:
AFAIK monkey logo styles got fixed in the navbar